### ISSUE TITLE
New Rule T0021: Use extension methods for Linq

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#9195]
Use extension methods for Linq

It should raise on Query syntax.
```
IEnumerable<int> numQuery1 = 
    from num in numbers
    where num % 2 == 0
    orderby num
    select num;
```

**Comment by pavel-mikula-sonarsource** [comment]
https://sonarsource.atlassian.net/browse/NET-1331
